### ISSUE TITLE
The addon has been renamed `storybook-addon-remix-react-router`

### ISSUE THREAD

**Issue by JesusTheHun** [JesusTheHun/storybook-addon-remix-react-router#63]
# TL;DR

```bash
npm uninstall storybook-addon-react-router-v6
npm i -D storybook-addon-remix-react-router
```

# Why
I created this package when the most popular router addon at the time didn't support `react-router@6.x`.
The name was picked to make it clear that this addon had support for the latest version.

Since then many things have changed ; the package grew in popularity and Remix took stewardship of `react-router`.

Clearly, I will not abandon this package if Remix releases a v7.
Since Storybook 8 is in RC and asked addon maintainers to make a new major release for it, I thought it was a nice opportunity to rename the package.

# What changes ?

Pretty much nothing.

The API does not change, the content of the package does not change.
The version number will be `3.x` to reflect the continuity of the project.

The only thing that changes is the name, and the fact that it targets Storybook 8 instead of Storybook 7.

The license has been changed to Apache 2.
